cluster-migration/src/main/java/com/wipro/ats/bdre/clustermigration/SourceStageLoad.java

```diff
@@ -0,0 +1,97 @@
+            {"stgBp", "stg-bp-cols", " Business partition columns"},
+            {"execId", "instance-exec-id", "Instance execution id"},
+            {"srcDb", "source-db", " Source database"},
+            {"srcTbl", "source-table", "Source table"},
+            {"filCon", "filter-condition", "Filter condition"},
+            {"ppid", "parent-process-id", "Parent Process Id"}
+    };
+
+    protected static Connection getHiveJDBCConnection(String dbName, String hiveConnection) throws Exception {
+        try {
+            Class.forName(IMConstant.HIVE_DRIVER_NAME);
+            Connection con = DriverManager.getConnection(hiveConnection + "/" + dbName, null, null);
+            con.createStatement().execute("set hive.exec.dynamic.partition.mode=nonstrict");
+            con.createStatement().execute("set hive.exec.dynamic.partition=true");
+            con.createStatement().execute("set hive.exec.max.dynamic.partitions.pernode=1000");
+            return con;
+        } catch (ClassNotFoundException e) {
+            throw new Exception(e);
+        } catch (SQLException e) {
+            throw new Exception(e);
+        }
+    }
+
+    public void execute(String[] params) throws Exception{
+
+        CommandLine commandLine = getCommandLine(params, PARAMS_STRUCTURE);
+        String sourceStageDb=commandLine.getOptionValue("src-stage-db");
+        String sourceStageTable = commandLine.getOptionValue("src-stg-table");
+        String stagePartitionCols = commandLine.getOptionValue("stg-part-cols");
+        String sourceRegCols=commandLine.getOptionValue("src-reg-cols");
+        String sourceBusinessPartitions = commandLine.getOptionValue("stg-bp-cols");
+        String instanceExecId = commandLine.getOptionValue("instance-exec-id");
+        String sourceDb=commandLine.getOptionValue("source-db");
+        String sourceTable = commandLine.getOptionValue("source-table");
+        String filterCondition = commandLine.getOptionValue("filter-condition");
+        String parentProcessId=commandLine.getOptionValue("parent-process-id");
+        String sourceHiveConnection = getSourceHiveConnection(parentProcessId,"hive-migration");
+
+        Connection conn = getHiveJDBCConnection(sourceDb,sourceHiveConnection);
+        Statement st = conn.createStatement();
+        String sourceStgLoadHQL="insert overwrite table "+sourceStageDb+"."+sourceStageTable+" partition("+stagePartitionCols+") select "+sourceRegCols+","+sourceBusinessPartitions+","+instanceExecId+" from "+sourceDb+"."+sourceTable+" where "+filterCondition;
+        st.executeUpdate(sourceStgLoadHQL);
+        st.close();
+        conn.close();
+        return ;
+    }
+    public String getSourceHiveConnection(String pid, String configGroup) {
+        GetProperties getProperties = new GetProperties();
+        java.util.Properties listForParams = getProperties.getProperties(pid, configGroup);
+        String sourceHiveConnection=listForParams.get("src-hive").toString();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "sourceHiveConnection". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1488)